### PR TITLE
Multi-facility copy updates

### PR DESCRIPTION
**Pull request by jessex**
## Description of the change

This finalizes all copy prior to the release of multi-facility support.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Related to #70 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/design-system/Tooltip.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ const TooltipContentsDiv = styled.div`
   border-radius: 5px;
   min-width: 20px;
   min-height: 20px;
+  max-width: 400px;
```
Comment (jessex) [inline src/design-system/Tooltip.tsx:27]: This ensure very long tooltip text doesn't just create an extremely long one-line tooltip across the entire page.

**Comment by jessex** [comment]
The lint issue pre-dates this PR so I'd rather skip it for now!

File: src/page-multi-facility/CreateBaselineScenarioPage.tsx [hunk 2]
```diff
@@ -47,11 +47,29 @@ const CreateBaselineScenarioPage: React.FC = () => {
       <ModalDialog open title="Welcome">
         <ModalContent>
           <WelcomeText>
-            Welcome to your new scenario. To get started, add in facility data
-            on the right-hand side of the page. Your initial scenario is also
-            your 'Baseline' - meaning this is where you should keep real-world
-            numbers about the current state of your facilities, their cases, and
-            mitigation steps.
+            This tool will help you model Covid-19 cases in a set of criminal
```
Comment (sprad) [inline src/page-multi-facility/CreateBaselineScenarioPage.tsx:50]: In most other places I've seen on Recidiviz's web site, Covid-19 is referred to in all caps like this: COVID-19.  My spellchecker even marks the lower case version as incorrect

File: src/page-multi-facility/CreateBaselineScenarioPage.tsx [hunk 3]
```diff
@@ -47,11 +47,29 @@ const CreateBaselineScenarioPage: React.FC = () => {
...
+            the real-world as possible. When you want to model something new (a
+            policy change, for instance), you'll be able to make a copy of your
+            baseline scenario, make the change there, and compare it to
+            baseline. (Note: Creating multiple scenarios isn't ready just yet,
+            but it will be available within the next few days.)
+            <br />
+            <br />
+            If you have any questions or concerns, feel free to drop us a note
+            at
+            <a href="mailto:covid@recidiviz.org"> covid@recidiviz.org</a>.
```
Comment (sprad) [inline src/page-multi-facility/CreateBaselineScenarioPage.tsx:72]: Nit: Leading space after the opening `a` tag

File: src/page-multi-facility/ScenarioSidebar.tsx [hunk 4]
```diff
@@ -52,19 +52,19 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
             }
             toggled={scenario?.dailyReports}
             label="Daily Reports"
-            labelHelp="Tooltip help Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+            labelHelp="If enabled, your baseline scenario will be shared with Recidiviz and CSG. This data will only be used to provide you with daily reports."
```
Comment (sprad) [inline src/page-multi-facility/ScenarioSidebar.tsx:55]: Unless "CSG" is an acronym that's well understood within this domain, you may want to consider spelling it out

File: src/page-multi-facility/ScenarioSidebar.tsx [hunk 5]
```diff
@@ -52,19 +52,19 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
...
             }
             toggled={scenario?.dataSharing}
             label="Data Sharing"
-            labelHelp="Tooltip help Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+            labelHelp="If enabled, your baseline scenario will be made available to Recidiviz and the research community to improve the model and the state of research on the spread of disease in facilities. Any public research will anonymize state and facility names."
           />
           <PromoBoxWithButton
             text={
-              "Turn on 'DailyReports' to receive Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+              "Turn on 'DailyReports' to receive daily analysis and status updates from Recidiviz and CSG."
```
Comment (sprad) [inline src/page-multi-facility/ScenarioSidebar.tsx:67]: Same CSG comment as above

File: src/page-multi-facility/CreateBaselineScenarioPage.tsx (hunk 2 above)
Comment (jessex) [reply src/page-multi-facility/CreateBaselineScenarioPage.tsx:50]: Good catch!

File: src/page-multi-facility/CreateBaselineScenarioPage.tsx (hunk 3 above)
Comment (jessex) [reply src/page-multi-facility/CreateBaselineScenarioPage.tsx:72]: That was by design to ensure there was a space between at and the email address, but I think I know a better way to do it.

File: src/page-multi-facility/ScenarioSidebar.tsx (hunk 4 above)
Comment (jessex) [reply src/page-multi-facility/ScenarioSidebar.tsx:55]: Fortunately it is!

**Pull request merged**